zephyr/Kconfig

```diff
@@ -27,8 +27,25 @@ config WEP
 config WPA_SUPP_CRYPTO
     bool "Crypto support for WiFi"
     select WEP
+    select MBEDTLS
+    select NRF_SECURITY
+    select MBEDTLS_LEGACY_CRYPTO_C
+    select MBEDTLS_ECP_ALL_ENABLED
```

Comment:
Just a question.. Do we need all ECP algorithms, or just `MBEDTLS_ECP_DP_SECP256R1_ENABLED` would be enough?

Reply:
You are right, we only need that, and that is the only curve which is enabled by default when we choose MBEDTLS_ECP_C, so, can remove this line altogether.